### ISSUE TITLE
Could not resolve argument "$containerConfigurator"

### ISSUE THREAD

**Issue by DennisdeBest** [markitosgv/JWTRefreshTokenBundle#243]
Hello,

I just upgraded this inside my Sylius 1.9 and it was working fine before but now I am getting the followin error: 

```
Could not resolve argument "$containerConfigurator" for "/srv/sylius/vendor/gesdinet/jwt-refresh-token-bundle/Depe  
  ndencyInjection/../Resources/config/services.php". You must typehint it (for example with "Symfony\Component\Depen  
  dencyInjection\Loader\Configurator\ContainerConfigurator" or "Symfony\Component\DependencyInjection\ContainerBuild  
  er").  
  ```
  
 Not sure what needs to be done to fix this but I'm going to have a look and keep it posted on here

**Comment by oschwartz** [comment]
Same problem with symfony 5.3

**Comment by mbabker** [comment]
This should have been fixed by https://github.com/markitosgv/JWTRefreshTokenBundle/pull/241 which is included in 0.12.0, make sure you're using the current version of this bundle.

(FWIW I've got a Symfony 5.3 app in production and haven't had this issue once that PR was merged)

**Comment by DennisdeBest** [comment]
Indeed it is fixed with version 0.12.0 just had to make sure I updated this bundle before the other ones. Thanks @mbabker I'll close the issue.
